frontend/src/metabase/query_builder/actions/querying.ts

```diff
@@ -166,8 +182,8 @@ export const CLEAR_QUERY_RESULT = "metabase/query_builder/CLEAR_QUERY_RESULT";
 export const clearQueryResult = createAction(CLEAR_QUERY_RESULT);
 
 export const QUERY_COMPLETED = "metabase/qb/QUERY_COMPLETED";
-export const queryCompleted = (question, queryResults) => {
-  return async (dispatch, getState) => {
+export const queryCompleted = (question: Question, queryResults: any) => {
```

Comment:
Let's not use `any`

Reply:
I wasn't able to find the correct shape of the query results and I saw any used in a different place